chain/chain/src/runtime/mod.rs

```diff
@@ -751,11 +751,10 @@ impl RuntimeAdapter for NightshadeRuntime {
                 storage_config.use_flat_storage,
             ),
         };
-        // We need to start recording reads if the stateless validation is
-        // enabled in the next epoch. We need to save the state transition data
-        // in the current epoch to be able to produce the state witness in the
-        // next epoch.
-        if checked_feature!("stable", StateWitnessSizeLimit, next_protocol_version)
+        // StateWitnessSizeLimit: We need to start recording reads if the stateless validation is
+        // enabled in the next epoch. We need to save the state transition data in the current epoch
+        // to be able to produce the state witness in the next epoch.
+        if checked_feature!("stable", StatelessValidationV0, next_protocol_version)
```

Comment:
once you are going over these, can you also use this `enabled` function for the check:
```
if StatelessValidationV0.enabled(next_protocol_version)
```

given that both map to the same check: 
https://github.com/near/nearcore/blob/b4435fb4ed90440206b53991406f7a6051d1531d/core/primitives-core/src/version.rs#L299

https://github.com/near/nearcore/blob/b4435fb4ed90440206b53991406f7a6051d1531d/core/primitives-core/src/version.rs#L270